blueocean-dashboard/src/main/js/components/Pipelines.jsx

```diff
@@ -43,14 +43,32 @@ export class Pipelines extends Component {
                 { organization }
             </Link> : '';
 
-        const headers = [
-            { label: translate('home.pipelineslist.header.name', { defaultValue: 'Name' }), className: 'name-col' },
-            translate('home.pipelineslist.header.health', { defaultValue: 'Health' }),
-            translate('home.pipelineslist.header.branches', { defaultValue: 'Branches' }),
-            translate('home.pipelineslist.header.pullrequests', { defaultValue: 'PR' }),
-            { label: '', className: 'actions-col' },
+        const labelName = translate('home.pipelineslist.header.name', { defaultValue: 'Name' });
+        const labelHealth = translate('home.pipelineslist.header.health', { defaultValue: 'Health' });
+        const labelBranches = translate('home.pipelineslist.header.branches', { defaultValue: 'Branches' });
+        const labelPullReqs = translate('home.pipelineslist.header.pullrequests', { defaultValue: 'PR' });
+
+        const columns = [ // TODO: resource labels
+            JTable.column(640, labelName, true),
+            JTable.column(70, labelHealth),
+            JTable.column(70, labelBranches),
+            JTable.column(70, labelPullReqs),
+            JTable.column(50, ''),
         ];
+
+        const pipelineRows = pipelines && pipelines.map(pipeline => {
+            const key = pipeline._links.self.href;
+            return (
+                <PipelineRowItem
+                    t={ translate }
+                    key={ key } pipeline={ pipeline }
+                    showOrganization={ AppConfig.showOrg() }
+                />
```

Comment:
I see `columns` was added as a propType for `PipelineRowItem` but not actually passed down here. Is it needed?